homework-12/main.js

```diff
@@ -0,0 +1,74 @@
+window.addEventListener('DOMContentLoaded', () => {
+    let firstNumber = '';
+    let secondNumber = '';
+    let signOperation = '';
+    let finish = false;
+    const digit = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '.'];
+    const action = ['-', '+', 'x', '/'];
+    const out = document.querySelector('.calc-screen p');
+
+    document.querySelector('.ac').addEventListener('click', () => {
+        out.textContent = 0;
+        firstNumber = '';
+        secondNumber = '';
+        signOperation = '';
+        finish = false;
+
+    });
+    document.querySelector('.buttons').addEventListener('click', (event) => {
```

Comment:
It should be one zero

https://user-images.githubusercontent.com/84010501/201359350-b7c5226c-2d26-46c0-9da8-dfcf9eb8ca0e.mov

I know that `%` and `+/-` buttons are not part of the task, but both should not impact the result
 
https://user-images.githubusercontent.com/84010501/201359729-1095e682-6034-4258-9dbc-983a39ab1379.mov

The screen output should contain all digits  


https://user-images.githubusercontent.com/84010501/201360119-e015d66f-b84d-4eac-95b0-041ad9a200e8.mov